src/MeasureTheory.jl

```diff
@@ -8,12 +8,12 @@ using MLStyle
 import TransformVariables
 const TV = TransformVariables
 
-using DistributionMeasures
+# using DistributionMeasures
 using TransformVariables: asℝ₊, as𝕀, asℝ, transform
 
 import Base
-import Distributions
-const Dists = Distributions
+# import Distributions
+# const Dists = Distributions
```

Comment:
This makes it looks like we don't depend on Distributions.jl. But we do, it's just that this code is now in the file `distributions.jl`
```suggestion
```